index.js

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 const program = require('commander');
 const { prompt } = require('inquirer');
-const { newSigningKey, signingKeyChangePW } = require('./src/cliActions');
-const { newSigningKeySchema, changePasswordSigningKeySchema } = require('./src/promtSchema');
+const { newSigningKey, signingKeyChangePW, dappBuildBundleFile } = require('./src/cliActions');
+const { newSigningKeySchema, changePasswordSigningKeySchema, buildBundleFileSchema } = require('./src/promtSchema');
```

Comment:
`buildBundleFileSchema` should be named `dappStreamSchema` think about it, you are not bundling the app. You are streaming it to a specific location